src/main/java/org/commcare/formplayer/beans/auth/FeatureFlagChecker.kt

```diff
@@ -0,0 +1,24 @@
+package org.commcare.formplayer.beans.auth
+
+/**
+ * Utility class to check for enabled previews and toggles
+ */
+object FeatureFlagChecker {
+
+    fun isPreviewEnabled(preview: String, userDetailsBean: HqUserDetailsBean): Boolean {
```

Comment:
Instead of passing in the user details here we could get it from the security context. Or use this utility that we already have: https://github.com/dimagi/formplayer/blob/61b49325e7a131e7ef78d63ca92d3ca413a6bbf2/src/main/java/org/commcare/formplayer/util/RequestUtils.java#L99-L106